encled

```diff
@@ -172,6 +174,16 @@ def main(argv):
             print_status(d[0], d[1], d[3])
         return(0)
 
+    if argv[1].upper() == 'GOOD' and argv[2].lower() == 'off':
+        for d in devlist:
+            if ((d[3][1]).upper()).replace(" ", "") != 'FAULT_ON' and argv[2].lower() != 'off':
```

Comment:
you don't need to check of 'off' here, because it's already in 'upper if'.

Reply:
right :)
modified.